README.md

```diff
@@ -55,3 +66,4 @@ for now just run each project by either following way
 | Shipment Command | http://localhost:8087/swagger-ui/index.html |
 | Shipment Handler | http://localhost:8088/swagger-ui/index.html |
 | Shipment Query   | http://localhost:8089/swagger-ui/index.html |
```

Comment:
It is not in this PR scope, but I think we can merge all the Swagger UI into a single one

Reply:
yeah, i open an new issue to do that. we can discuss in detail under that issue.